cdap-docs/developers-manual/source/metadata/index.rst

```diff
@@ -1,23 +1,31 @@
 .. meta::
     :author: Cask Data, Inc.
-    :copyright: Copyright © 2015 Cask Data, Inc.
+    :copyright: Copyright © 2015-2017 Cask Data, Inc.
 
-.. _metadata-lineage:
+:hide-toc: true
+
+.. _metadata:
+
+========
+Metadata
+========
+
+.. toctree::
+   :maxdepth: 1
+   
+    CDAP Metadata UI <metadata-ui>
```

Comment:
This anchor is wrong? It should be overview? Because of this, the CDAP Metadata UI appears as the first link in the table of contents for this section, and seems wrong